qdrant-landing/content/documentation/tutorials/async-api.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Asynchronous API
+weight: 18
+---
+
+# Using Qdrant asynchronously
+
+Databases are often launched as separate services and are accessed via a network. All the interactions with them are IO-bound and can 
+be performed asynchronously so as not to waste time actively waiting for a server response. If you use Python, that is achieved by 
+using `async/await` syntax. That lets the interpreter switch to another task while waiting for a response from the server.
+
+Qdrant exposes two interfaces: HTTP and gRPC. The official SDKs for different languages are based on the autogenerated clients, and
+in Python [qdrant-client](https://github.com/qdrant/qdrant-client), you can call all the methods asynchronously. This tutorial presents 
+how to do it.
+
+## When the usage of async API is justified?
+
+If the application you are writing will never support multiple users at once, for example, it is a script run once daily, then there 
+is no need to use async API. But if you are writing a web service that multiple users will use simultaneously, you shouldn't be 
+blocking the threads of the web server as it limits the number of concurrent requests it can handle. In this case, you should use 
+the async API.
+
+Modern web frameworks like [FastAPI](https://fastapi.tiangolo.com/) and [Quart](https://quart.palletsprojects.com/en/latest/) support 
+async API out of the box. Asynchronous code cannot be incorporated into an existing synchronous codebase, so it's better to choose 
+the proper tool from the start.
+
+## How to use async API?
+
+As usual, calling any method of Qdrant requires establishing a connection to the server. We need to create an instance of `QdrantClient`
+that will act as a gateway. If you want to do it locally, please make sure Qdrant server is running. If it's not, then you can launch it
+in a Docker container:
+
+```bash
+docker run -p "6333:6333" -p "6334:6334" qdrant/qdrant:v1.4.0
```

Comment:
Actually, is there a particular reason why latest is not ok for this tutorial?

Reply:
Changed to latest